src/playbacks/html5_video/html5_video.js

```diff
@@ -261,19 +309,8 @@ export default class HTML5Video extends Playback {
     })
   }
 
-  checkBufferState(bufferedPos) {
-    var playbackPos = this.el.currentTime + 0.05; // 50 ms threshold
-    if (this.isPlaying() && playbackPos >= bufferedPos) {
-      this.trigger(Events.PLAYBACK_BUFFERING, this.name)
-      this.buffering = true
-    } else if (this.buffering) {
-      this.trigger(Events.PLAYBACK_BUFFERFULL, this.name)
-      this.buffering = false
-    }
-  }
-
   typeFor(src) {
-    return (src.indexOf('.m3u8') > 0) ? 'application/vnd.apple.mpegurl' : 'video/mp4'
```

Comment:
Oh yeh. I was thinking the `hls` playback was still the one that would be used. Will put that back